api/elections.go

```diff
@@ -600,3 +610,63 @@ func getElection(electionID []byte, vs *state.State) (*models.Process, error) {
 	}
 	return process, nil
 }
+
+// POST /elections/filter/page/<page>
+// returns a paginated list of elections filtered by partial organizationID, partial processID,
+// process status and with results available or not
+func (a *API) electionFilterPaginatedHandler(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
+	// get organizationId from the request body
+	body := struct {
+		OrganizationID types.HexBytes `json:"organizationId"`
+		ElectionID     types.HexBytes `json:"electionId"`
+		WithResults    bool           `json:"withResults"`
+		Status         string         `json:"status"`
+	}{}
+	if err := json.Unmarshal(msg.Data, &body); err != nil {
+		return fmt.Errorf("%w (%x): %v", ErrMarshalingServerJSONFailed, msg.Data, err)
```

Comment:
here you would use Withf instead of WithErr

`return ErrMarshalingServerJSONFailed.Withf("(%x): %v", msg.Data, err)`